### PR TITLE
docs: update HACKING doc with latest information

### PR DESCRIPTION
**Pull request by sinnykumari**
Like latest MCO needs golang 1.13

### CODE REVIEW

File: docs/HACKING.md [hunk 1]
```diff
@@ -66,7 +65,7 @@ All tests (unit and e2e) can be executed with:
 
 Dependencies are managed with [go modules](https://github.com/golang/go/wiki/Modules) but committed directly to the repository. 
 
-Please ensure that you are using Go v1.12.  Using a differnt version may result in unexpected behavior.
+Please ensure that you are using Go v1.13.  Using a differnt version may result in unexpected behavior.
```
Comment (kikisdeliveryservice) [inline docs/HACKING.md:68]: typo on different

Comment (sinnykumari) [reply docs/HACKING.md]: thanks, fixed.

**Comment by kikisdeliveryservice** [comment]
(just want to see the state of this test)

/test e2e-gcp-upgrade

**Pull request merged**